test/e2e/node/pods.go

```diff
@@ -179,12 +179,13 @@ var _ = SIGDescribe("Pods Extended", func() {
 		ginkgo.BeforeEach(func() {
 			podClient = f.PodClient()
 		})
+
 		/*
 			Release : v1.9
 			Testname: Pods, QOS
-			Description:  Create a Pod with CPU and Memory request and limits. Pos status MUST have QOSClass set to PodQOSGuaranteed.
+			Description:  Create a Pod with CPU and Memory request and limits. Pod status MUST have QOSClass set to PodQOSGuaranteed.
 		*/
-		framework.ConformanceIt("should be submitted and removed ", func() {
+		framework.ConformanceIt("should be set on Pods with matching resource requests and limits for memory and cpu", func() {
```

Comment:
If we're going to change the description, it might as well make sense. What should be set on Pods?

Reply:
In this case, "Pods Set QOS Class".  In the full test name (Describe + Context etc.) as used by the generated conformance docs this would show up (see https://github.com/cncf/k8s-conformance/blob/master/docs/KubeConformance-1.11.md#pods-set-qos-class-should-be-submitted-and-removed).  Perhaps the `KubeDescribe` above should use `Pod QOS Class` instead

The "normalized" test name that ends up in conformance.txt ends up just being the `It` with tags stripped.  This is why [I'm wondering if we want to change what we put in that list](https://github.com/kubernetes/kubernetes/pull/78710#issuecomment-499184756)